types/types.go

```diff
@@ -12,6 +18,71 @@ type EntityID = []byte
 
 type CensusRoot = []byte
 
+// VoterID is the indentifier of a voter, it is represented as random bytes
+// The first byte of the slice indicates one of the supported identifiers
+// For example for an Ethereum public key the VoterID is [1, pubkb0, pubkb1, ...]
+// where pubkb0 is the first byte of the Ethereum public key
+type VoterID []byte
+
+// VoterIDType represents the type of a voterID
+type VoterIDType = uint8
+
+const (
+	VoterID_Type_Undefined VoterIDType = 0
+	VoterID_Type_ECDSA     VoterIDType = 1
+)
+
+// Enum value maps for VoterIDType.
+var (
+	VoterID_Type_name = map[VoterIDType]string{
```

Comment:
I think we don't need to export this map. There are helper methods on the VoterID type that make the job.